web/package.json

```diff
@@ -104,6 +106,10 @@
     "unit-test": "mocha --require @babel/register --reporter spec ./test/unit-test",
     "unit-test-watch": "mocha -w --require @babel/register --reporter spec ./test/unit-test",
     "test": "node/node test/mocha-chrome-wrapper.js ${MOCHA_URL:=http://localhost:7021/koski/test/runner.html}?grep=${MOCHA_GREP} --chrome-flags '[\"--use-mock-keychain\"]'",
+    "test:e2e": "npx playwright test",
```

Comment:
Vähän hämää tämä e2e-termin käyttö, koska nämä ei oikeasti ole end-to-end -testejä, koska niin suuri osa muista palveluista on mockailtu. En kyllä heti keksi, mikä olisi parempi termi.

Reply:
Jos rinnalla ei ajettaisi myös mocha-testejä, voisi tossa käyttää pelkkää "test"-komentoa. Esim. `test:playwright` olisi vähän parempi.

Reply:
Nyt näin: 
```json
{ 
    "playwright:test": "npx playwright test",
    "playwright:test:debug": "npx playwright test --headed",
    "playwright:test:trace": "npx playwright test --trace on && npx playwright show-report",
    "playwright:codegen": "npx playwright codegen"
}
```